2014-01-02 18:49:27 - ERROR - SaveFTP::send() Exception: [Errno 2] No such file or directory: 'aa.py0'
2014-01-02 19:24:26 - INFO - dumpMySQL::dump() cmd: mysqldump -hlocalhost -ulok -p123 scrapy > /tmp/scrapy_20140102192426.sql
2014-01-02 19:33:52 - INFO - dumpMySQL::dump() cmd: mysqldump -hlocalhost -ulok -p123 scrapy > scrapy_20140102193352.sql
2014-01-02 19:35:29 - INFO - dumpMySQL::dump() cmd: mysqldump -hlocalhost -ulok -p123 scrapy > scrapy_20140102193529.sql
2014-01-02 19:36:53 - INFO - dumpMySQL::dump() cmd: mysqldump -hlocalhost -ulok -p123 scrapy > scrapy_20140102193653.sql
2014-01-02 19:36:53 - ERROR - SaveFTP::login() Exception: [Errno 111] Connection refused
2014-01-02 19:36:53 - ERROR - SaveFTP::send() Exception: 'NoneType' object has no attribute 'cwd'
2014-01-02 19:37:46 - INFO - dumpMySQL::dump() cmd: mysqldump -hlocalhost -ulok -p123 scrapy > scrapy_20140102193746.sql
